streambed-machine/README.md

```diff
@@ -0,0 +1,3 @@
+# Streambed Simplified
+
+Simplified interface to streambed CommitLog
```

Comment:
Hmm. That implies streambed is complex. ;-) How about "Streambed integration".

Also needs beefing up.